lighthouse-core/fraggle-rock/gather/session.js

```diff
@@ -20,9 +20,9 @@ class ProtocolSession {
     const originalEmit = session.emit;
     // @ts-expect-error - Test for the monkeypatch.
     if (originalEmit[SessionEmitMonkeypatch]) return;
-    session.emit = (method, ...params) => {
-      originalEmit.call(session, '*', {method, params});
-      return originalEmit.call(session, method, ...params);
+    session.emit = (method, ...args) => {
+      originalEmit.call(session, '*', {method, params: args[0]});
```

Comment:
this version mismatched the driver version and was only excercised in tests.

it will be going away imminently when I can upgrade to latest puppeteer that doesn't need this monkeypatch anymore